pallets/orml-currencies-allowance-ext/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+authors = ["Pendulum Chain"]
+edition = "2021"
+name = "orml-currencies-allowance-ext"
+version = "1.0.0"
+
+[dependencies]
+codec = {package = "parity-scale-codec", version = "3.1.5", default-features = false, features = ["derive", "max-encoded-len"]}
+scale-info = {version = "2.2.0", default-features = false, features = ["derive"]}
+serde = {version = "1.0.130", default-features = false, features = ["derive"], optional = true}
+sha2 = {version = "0.8.2", default-features = false}
+
+# Substrate dependencies
+frame-support = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37", default-features = false}
+frame-system = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37", default-features = false}
+sp-core = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37", default-features = false}
+sp-runtime = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37", default-features = false}
+sp-std = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37", default-features = false}
+
+orml-tokens = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git", default-features = false, branch = "polkadot-v0.9.37" }
+orml-currencies = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git", default-features = false, branch = "polkadot-v0.9.37" }
+orml-traits = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git", default-features = false, branch = "polkadot-v0.9.37" }
+[dev-dependencies]
```

Comment:
Maybe add a new line above `[dev-dependencies` for better readability

Reply:
done